### ISSUE TITLE
New apps using latest Mobile SDK are no longer accepted by Apple because of UIWebView usage

### ISSUE THREAD

**Issue by piotr-marat** [dotpay/Mobile-SDK-iOS#15]
ITMS-90809: Deprecated API Usage - New apps that use UIWebView are no longer accepted. Instead, use WKWebView for improved security and reliability. Learn more (https://developer.apple.com/documentation/uikit/uiwebview).

**Comment by adamovitch** [comment]
Hi! I faced the same problem. Do you plan to replace WebView?

**Comment by dotM4rio** [comment]
Yes, there's a plan to provide an updated SDK for ios. Version 2.0 will be available in about two weeks

**Comment by chdo002** [comment]
> Yes, there's a plan to provide an updated SDK for ios. Version 2.0 will be available in about two weeks

still not updated

**Comment by chdo002** [comment]
> > Yes, there's a plan to provide an updated SDK for ios. Version 2.0 will be available in about two weeks
> 
> still not updated

my app has been rejected for UIWebView in your SDK， please hurry！

**Comment by chdo002** [comment]
> > > Yes, there's a plan to provide an updated SDK for ios. Version 2.0 will be available in about two weeks
> > 
> > 
> > still not updated
> 
> my app has been rejected for UIWebView in your SDK， please hurry！

I have given up this SDK and  removed it, has your company closed down?

**Comment by adamovitch** [comment]
> > > > Yes, there's a plan to provide an updated SDK for ios. Version 2.0 will be available in about two weeks
> > > 
> > > 
> > > still not updated
> > 
> > 
> > my app has been rejected for UIWebView in your SDK， please hurry！
> 
> I have given up this SDK and removed it, has your company closed down?

The same story. I change it to Stripe. It's a little bit complicate for integration, but nobody can 
wait several months

**Comment by dotM4rio** [comment]
@adamovitch @chdo002 @piotr-marat 
New version 2.0 without UIWebView has been released https://github.com/dotpay/Mobile-SDK-iOS/releases/tag/2.0.0
